collection_requirements.rst

```diff
@@ -0,0 +1,145 @@
+*****************************
+Ansible Collections Checklist
+*****************************
+
+.. contents:: Topics
+
+Overview
+========
+
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
+
+Please raise feedback by
+
+* Discussing in ``#ansible-community`` in Freenode IRC
+* Adding to the `Community Working Group IRC meeting <https://github.com/ansible/community/issues/539>`_
+* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ against this repo
+
+Be sure you're subscribed to:
+* `Changes impacting Collections <https://github.com/ansible-collections/overview/issues/45>` to track changes that Collection maintainers should be aware of 
+* The Bullhorn, a newsletter for the Ansible developer community, `back issues and how to add content <https://github.com/ansible/community/issues/546>`_
+
+Why is this needed
+===================
+
+
+Repo structure
+===============
+
+galaxy.yml
+----------
+
+* tags MUST be set
+* dependencies MUST be set to ``'>=1.0.0'`` (Might have to think about dependency order when publishing)
+
+meta/runtime.yml
+----------------
+Example: `meta/runtime.yml <https://github.com/ansible-collections/collection_template/blob/main/meta/runtime.yml>`_
+* MUST define the minimim version of Ansible which this collection works with
+  * If the Collection works with Ansible 2.9, then this should be set to `>=2.9.10`
+
+Modules & Plugins
+------------------
+
+Documentation
+~~~~~~~~~~~~~~
+All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
+
+* Use the FQCN for ``M(...)`` See `Linking within module documentation<https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
+* Use collection version numbers for `version_added` (otherwise `version_added_collection` must be provided, but this is discouraged)
+
+All module and plugin ``EXAMPLES`` MUST:
+
+* Use FQCN for module (or plugin) name.
+* for modules (or plugins) left in ansible-base use ``ansible.builtin.template``
+
+Other items:
+* You MUST Use the FQCN for ``extends_documentation_fragment:``, unless you are referring to doc_fragments from ansible-base
+
```

Comment:
```suggestion
* You MUST use your collection version that is included in the 2.10 Ansible package for the `version_added` field for anything added in 2.10. This field reflects the collection version now, not the ansible-base version.
```

Reply:
That's not entirely true. You need to use the version number of the collection where the plugin / module / option was added, which is <= to the version of the collection inluded in 2.10.0. Depending on how many versoins of the collection have been released before ansible 2.10.0 will be released, there can be several different values. (In c.g, both the values 0.2.0 and 1.0.0 will appear, and maybe 1.1.0 if that's released in time to make it into 2.10.0.)

Also see https://github.com/ansible-collections/overview/pull/90/files#diff-31dcce2efaaf88ba4f5a7824b787cb9eR55 which already mentions `version_added`.

Reply:
oh hah didn't see that! Will delete my comment.